demisto_sdk/commands/common/errors.py

```diff
@@ -1527,6 +1528,10 @@
 }
 
 
+if is_external_repository():
+    ALLOWED_IGNORE_ERRORS = [err["code"] for err in ERROR_CODE.values()]
```

Comment:
shouldn't this be done when we initialize it anyway? 
`ALLOWED_IGNORE_ERRORS = ... if is_external_repo() else ...`